private-templates-service/client/src/components/Common/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -12,10 +12,11 @@ interface Props {
   onClick?: () => void;
   cardtemplate: Template;
   templateVersion: string;
+  hostConfig?: any;
```

Comment:
im guessing we can't find a type for this?

Reply:
yep, in fact the type definition for `hostConfig` in the `adaptiveCards` library is `?: any`